gradle/checkstyle.gradle

```diff
@@ -0,0 +1,13 @@
+apply plugin: 'checkstyle'
+
+checkstyle {
+    toolVersion = '8.1'
+    ignoreFailures = false
```

Comment:
You can override default configuration path so that findbugs.xml could be somewhere else in project (for example next to this file)